6.4.0/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM alpine:3.10
+
+RUN apk update \
+    && apk upgrade --no-cache
+RUN apk add --no-cache bash=5.0.0-r0
+RUN apk add --no-cache curl=7.66.0-r0
+
+ADD https://github.com/concourse/concourse/releases/download/v6.4.0/fly-6.4.0-linux-amd64.tgz /tmp/fly.tgz
```

Comment:
`ADD` is generally discouraged as it leads to larger images, though the archive file here is relatively small (~20MB). Try combining the retrieval and extraction into a single step (the intermediary file written to disk is not really necessary and can be replaced with an anonymous pipe, and the `chmod` command can also be dropped if we preserve file permissions using the `-p` flag to `tar`):

```
RUN curl -L https://github.com/concourse/concourse/releases/download/v6.4.0/fly-6.4.0-linux-amd64.tgz \
    | tar -pzxv -C /usr/bin 
```